### PR TITLE
Use gpg to unlock ansible vault

### PR DESCRIPTION
**Pull request by anadahz**
- Add gpg related documentation and update commands
- Create gpg decrypt script
- Add public key for user andz

### CODE REVIEW

**Comment by anadahz** [comment]
@francisco-core Please add your gpg public key.

**Comment by francisco-core** [comment]
tested decrypting with my key. Good to merge. This is pretty epic. Thanks @anadahz

**Comment by anadahz** [comment]
@francisco-core Thank you for the review. Merging!

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ placed under the `host_vars` directive for the specified host,
 This role is used to add/remove users, groups, permissions rights and access to
 hosts. You can deploy this role by running:
 
-`cd ansible && ansible-playbook --vault-id @prompt -i inventory/production deploy-admin.yml`
+`cd ansible && ansible-playbook -i inventory/production deploy-admin.yml`
 
 #### Add a user to all hosts
```
[the remark this answers is not in the document]
Comment (anadahz) [reply README.md:60]: It's the correct heading as it is refers to the admin role.

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ To generate the SSH public key you should use the `ansible-vault`
 (encryption/decryption utility for Ansible data files), an example command looks
 like:
 
-`ansible-vault encrypt_string --vault-id @prompt 'ssh-ed25519 XXX' --name sshkey`
+`ansible-vault encrypt_string 'ssh-ed25519 XXX' --name sshkey`
```
[the remark this answers is not in the document]
Comment (anadahz) [reply README.md:73]: The `XXX` is an example for an SSH key.

**Pull request merged**